### PR TITLE
Implement PostgresClient.get_df using COPY

### PR DESCRIPTION
**Pull request by ig248**
Streaming results as CSV to STDOUT

## Example

```python
import tentaclio as tio

query = "SELECT * FROM table LIMIT 1000000"
url = tio.api.authenticate("postgresql://hostname/db_name")

%%time  # Wall time: 113s
with tio.clients.sqla_client.SQLAlchemyClient(url) as c:
    df = c.get_df(query)

%%time  # Wall time: 51s
with tio.clients.postgres_client.PostgresClient(url) as c:
    df = c.get_df(query)
```

### CODE REVIEW

**Comment by ig248** [comment]
@capitancambio the functional test exists and has failed successfully :) fixing now

File: CHANGELOG.txt [hunk 1]
```diff
@@ -6,7 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.0.1-alpha.6] - 2020-03-18
 ### Added
- -  Allow ftp scandir fallback to dir when mlst is not implemented on the server
+ - Allow ftp scandir fallback to dir when mlst is not implemented on the server
+### Changed
+ - Use `COPY` in `get_df` for faster Postgres queries
```
Comment (capitancambio) [inline CHANGELOG.txt]: that has been already released. please add a new one

File: tests/functional/postgres/test_api.py [hunk 2]
```diff
@@ -36,12 +36,12 @@ def fixture_df():
...
-        f"postgresql://hostname/tentaclio-db::{TEST_TABLE_NAME}", mode="w"
+        f"postgresql://hostname{fixture_client.url.path}::{TEST_TABLE_NAME}", mode="w"
     ) as writer:
         fixture_df.to_csv(writer, index=False)
 
     with clients.PostgresClient(
-        credentials.authenticate("postgresql://hostname/tentaclio-db")
+        credentials.authenticate(f"postgresql://hostname{fixture_client.url.path}")
     ) as client:
         retrieved_df = client.get_df(f"select * from {TEST_TABLE_NAME}")
```
Comment (capitancambio) [inline tests/functional/postgres/test_api.py]: would be grand if you checked that the data is there...

File: CHANGELOG.txt [hunk 3]
```diff
@@ -4,14 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.1-alpha.8] - 2020-03-19
+### Added
+ - `PostgresClient.get_df_unsafe()` using COPY for faster Postgres queries
+
```
Comment (capitancambio) [inline CHANGELOG.txt]: please bump the `setup.py` too please

**Pull request merged**